scripts/functional_tests.js

```diff
@@ -17,12 +17,23 @@
  * under the License.
  */
 
-require('../src/setup_node_env');
-require('@kbn/test').runTestsCli([
+// eslint-disable-next-line no-restricted-syntax
```

Comment:
Would all of these changes feel better as a separate file (e.g. `scripts/functional_tests_for_coverage.js` or similar)? You'd be able to skip the list combining and eslint disables... I can't decide, just a thought

Reply:
It was in a separate file initially, but I talked to Spencer and made it this. @spalger is ok or better in an own file?